tier2_standards/test_coverage.md

```diff
@@ -4,14 +4,61 @@ This is one of the tier 2 standards. See full list [on the overview page](README
 
```

Comment:
This one is really thorough!  And I think it could be a good resource even outside of the INS JWST tools 👏

Reply:
Thanks! I just kept thinking of things to add so it got kinda long... aw well!